docs/boards/nrf52840_dongle.md

```diff
@@ -0,0 +1,73 @@
+# <img alt="OpenSK logo" src="../img/OpenSK.svg" width="200px">
+
+## Nordic nRF52840 Dongle
+
+![Nordic dongle](../img/dongle_front.jpg)
+
+### Flashing using DFU
```

Comment:
Maybe putting `(preferred method)` in the title as the other ways aren't tested anymore since DFU support was added.

Reply:
Sounds good, done.